### PR TITLE
Native setup

### PR DESCRIPTION
**Pull request by ykotturi**
This branch creates a React Native application via Expo. It also includes two drafted views for Peerdea: the Home screen and Create New Group screen, and basic navigation support between screens as well as a few other boilerplate screens provided by start up.

image

image

### CODE REVIEW

File: peerdea-native/screens/CreateGroupScreen.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
```
Comment (machang19) [inline peerdea-native/screens/CreateGroupScreen.js]: This File seems like it should be broken up into three files for the two react components and the css

File: peerdea-native/screens/HomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  };
+
+  _handleHelpPress = () => {
+    WebBrowser.openBrowserAsync(
+      'https://docs.expo.io/versions/latest/guides/up-and-running.html#can-t-see-your-changes'
+    );
+  };
+}
+
+const styles = StyleSheet.create({
```
Comment (machang19) [inline peerdea-native/screens/HomeScreen.js]: Again css should probably be in separate file but not a big deal

File: peerdea-native/navigation/MainTabNavigator.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import HomeScreen from '../screens/HomeScreen';
+import LinksScreen from '../screens/LinksScreen';
+import CreateGroupScreen from '../screens/CreateGroupScreen';
+import JoinGroupScreen from '../screens/LinksScreen';
+import SettingsScreen from '../screens/SettingsScreen';
+
+// HomeStack follows a similar pattern as what is provided here: http://facebook.github.io/react-native/docs/navigation
+const HomeStack = createStackNavigator({
+  Home: {screen: HomeScreen},
+  CreateGroup: {screen: CreateGroupScreen},
```
Comment (machang19) [inline peerdea-native/navigation/MainTabNavigator.js]: Is this how we get  onPress={() => navigate('CreateGroup')} to navigate to CreateGroupScreen?

**Comment by machang19** [comment]
+1

File: peerdea-native/screens/CreateGroupScreen.js (hunk 1 above)
Comment (ykotturi) [reply peerdea-native/screens/CreateGroupScreen.js]: Actually, for React Native, it seems it's commonplace to store stylesheet in the same file as component: https://freecontent.manning.com/applying-and-organizing-styles-in-react-native/

File: peerdea-native/navigation/MainTabNavigator.js (hunk 3 above)
Comment (ykotturi) [reply peerdea-native/navigation/MainTabNavigator.js]: Yes - from what I've learned so far, this section of code is kind of like what a routes.js would do in regular React

File: peerdea-native/screens/CreateGroupScreen.js (hunk 1 above)
Comment (ykotturi) [reply peerdea-native/screens/CreateGroupScreen.js]: Oh, for the other half of your comment: one component to one file, I'm happy to strive towards that from here on out. Good point!

**Pull request merged**